htslib/kstring.h

```diff
@@ -154,6 +204,13 @@ static inline char *ks_release(kstring_t *s)
 	return ss;
 }
 
+/// Safely free the underlying buffer in a kstring.
+static inline void ks_free(kstring_t *s)
+{
+    if (s) free(s->s);
```

Comment:
This function is now in two minds whether `s!=NULL` is a precondition. Either nuke `if (s)` or put the `ks_initialize` inside the if too.

Reply:
Should be fixed now.

Reply:
Thanks :+1: